cmd/server/app/migrate_up.go

```diff
@@ -127,10 +130,64 @@ var upCmd = &cobra.Command{
 			return fmt.Errorf("error preparing authz client: %w", err)
 		}
 
-		return nil
+		store := db.NewStore(dbConn)
+		return migrateProjectsRootAndMetadata(ctx, cmd, store)
 	},
 }
 
+func migrateProjectsRootAndMetadata(ctx context.Context, cmd *cobra.Command, store db.Store) error {
+	cmd.Println("Migrating projects to have no parent...")
+	projs, err := store.ListNonOrgProjects(ctx)
+	if err != nil {
+		return fmt.Errorf("error while listing non-org projects: %w", err)
+	}
+
+	for _, p := range projs {
+		m := projects.NewSelfEnrolledMetadata()
+		if err := json.Unmarshal(p.Metadata, &m); err != nil {
+			return fmt.Errorf("error while marshalling old metadata: %w", err)
+		}
```

Comment:
This pattern feels weird (and it's ignoring the parent's metadata entirely).  If we want to set the self-enrolled bit, it feels like we should do so _after_ unmarshalling (unless the goal is to have a default, and overwrite it if the value is set in the JSON, but that probably deserves a comment).

Reply:
We didn't have anything relevant in the metadata. this starts defining _some_ structure in the metadata.